odl/space/npy_tensors.py

```diff
@@ -283,17 +283,22 @@ def __init__(self, shape, dtype=None, order='A', **kwargs):
         elif inner is not None:
             self.__weighting = NumpyTensorSpaceCustomInner(inner)
         else:
-            self.__weighting = NumpyTensorSpaceNoWeighting(exponent)
+            # No weighting, i.e., weighting with constant 1.0
+            self.__weighting = NumpyTensorSpaceConstWeighting(1.0, exponent)
+
+        # Make sure there are no leftover kwargs
+        if kwargs:
+            raise TypeError('got unknown keyword arguments {}'.format(kwargs))
 
     @property
     def impl(self):
         """Name of the implementation back-end: ``'numpy'``."""
         return 'numpy'
 
     @property
-    def exponent(self):
-        """Exponent of the norm and the distance."""
-        return self.weighting.exponent
+    def default_order(self):
+        """Default storage order for new elements in this space: ``'C'``."""
+        return 'C'
```

Comment:
no way to select this?

Reply:
I cannot find an obvious way, but I think it's safe to say that Numpy will use C ordering by default for a while.